README.md

```diff
@@ -12,6 +12,74 @@ The UCRM Plugin SDK can be installed with [Composer](https://getcomposer.org/).
 composer require ubnt/ucrm-plugin-sdk
 ```
 
+## Usage
+Simple example using available SDK classes:
+```php
+require_once __DIR__ . '/vendor/autoload.php';
+
+// Get UCRM log manager.
+$log = \Ubnt\UcrmPluginSdk\Service\PluginLogManager::create();
+
+// Check if there is a user logged into UCRM and has permission to view invoices.
+// https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/docs/security.md
+$security = \Ubnt\UcrmPluginSdk\Service\UcrmSecurity::create();
+$user = $security->getUser();
+if (
+    ! $user
+    || ! $user->hasViewPermission(\Ubnt\UcrmPluginSdk\Security\PermissionNames::BILLING_INVOICES)
+) {
+    if (! headers_sent()) {
+        header("HTTP/1.1 403 Forbidden");
+    }
+
+    $log->appendLog('Someone tried to access page only for admins with permission to view invoices.');
+
+    die('You\'re not allowed to access this page.');
+}
+
+$log->appendLog('Starting invoice export.');
+
+// Get export format from plugin's configuration.
+// https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/docs/file-structure.md#dataconfigjson
```

Comment:
This is unclear - are we assuming here that there's an `exportFormat` setting, coming from the manifest?

Reply:
Yes we are and it's supposed to be unclear as it's just example usage of the SDK. You can get more info at the link in the comment.